cmd/restic/cmd_backup.go

```diff
@@ -60,11 +60,11 @@ Exit status is 3 if some source data could not be read (incomplete snapshot crea
 		t.Go(func() error { term.Run(t.Context(globalOptions.ctx)); return nil })
 
 		err := runBackup(backupOptions, globalOptions, term, args)
-		if err != nil {
-			return err
-		}
 		t.Kill(nil)
-		return t.Wait()
+		if err := t.Wait(); err != nil {
+			debug.Log("term.Run() returned err: %v", err)
```

Comment:
As far as I can see `t.Wait()` always returns nil. So please change this to panic if a non-nil error is returned. Also please use `werr` or similar as variable name instead of shadowing `err`.